README-windows.md

```diff
@@ -19,21 +19,16 @@
     1. Open a command prompt and navigate to the directory with the vendor-sdk repo.
     2. Run the following commands:
 
-            mkdir build
-            cd build
-            cmake -G "MSYS Makefiles" ..
-            make
+            cmake -S . -B build -G "MSYS Makefiles" ..
+            cmake --build build
 
-    3. You should now have a `libelectionguard.a`.
+    3. You should now have a `electionguard.a` or `electionguard.dll` (depending on the how cmake was configured).
```

Comment:
The commands listed in Step 2 should only build an artifact (`.a`). 
It would only build a .dll if `-DBUILD_SHARED_LIBS=ON` is passed with command.